benchmarks/benchmark/tools/locust-load-inference/sample-tfvars/tgi-sample.tfvars

```diff
@@ -2,17 +2,19 @@ credentials_config = {
   fleet_host = "https://connectgateway.googleapis.com/v1/projects/$PROJECT_NUM/locations/global/gkeMemberships/ai-benchmark"
 }
 
-project_id = "change-me"
+project_id = "$PROJECT_ID"
 
 namespace = "benchmark"
 ksa       = "benchmark-ksa"
 
+k8s_hf_secret = "hf-token"
+
 # Locust service configuration 
 artifact_registry                        = "us-central1-docker.pkg.dev/$PROJECT_ID/ai-benchmark"
 inference_server_service                 = "tgi" # inference server service name
 locust_runner_kubernetes_service_account = "sample-runner-ksa"
 output_bucket                            = "benchmark-output"
-gcs_path                                 = "gs://${PROJECT_ID}-ai-gke-benchmark-fuse/ShareGPT_V3_unfiltered_cleaned_split_filtered_prompts.txt"
+gcs_path                                 = "gs://ai-gke-benchmark-fuse/ShareGPT_V3_unfiltered_cleaned_split_filtered_prompts.txt"
```

Comment:
Did we make this a fully public bucket that anyone can use?

Reply:
This is not a public bucket, I'm not sure we want to host one yet. I addressed my goals in the comment below :) let's continue the conversation there